gaphor/ui/tests/test_handletool.py

```diff
@@ -9,7 +9,7 @@
 from gi.repository import Gdk, Gtk
 
 from gaphor import UML
-from gaphor.application import Application
+from gaphor.application import Session
```

Comment:
I think the import above of unittest can be removed now that you completed the conversion on this module.